react/features/conference/components/Conference.native.js

```diff
@@ -232,12 +259,41 @@ class Conference extends Component {
 }
 
 /**
+ * Maps dispatching of some action to React component props.
+ *
+ * @param {Function} dispatch - Redux action dispatcher.
+ * @private
+ * @returns {{
+ *     _onConnect: Function,
+ *     _onDisconnect: Function,
+ *     _onToolbarVisibilityChange: Function
+ * }}
+ */
+function _mapDispatchToProps(dispatch) {
+    return {
+        _onConnect() {
```

Comment:
These are calling dispatch with action creators which may or may not return an asynchronous action so I see no way of saying whether and how much the whole thing is asynchronous or synchronous. I'm OK with the names. If you'd like to apply some other naming scheme, I'd rather we talk it in a meeting and I'll go and apply it outside this PR.